bindings/wasm/examples/worker.ts

```diff
@@ -12,15 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Document, Material, Node, WebIO} from '@gltf-transform/core';
-import {KHRMaterialsUnlit, KHRONOS_EXTENSIONS} from '@gltf-transform/extensions';
+import { Document, Material, Node, WebIO } from '@gltf-transform/core';
```

Comment:
Can we figure out what's going on with our clang formats? It's impossible to read these diffs if they're full of white space corrections. Are you using VSCode? I have a `.clang-format` file in our root, so ideally that should be standardizing this. I should really figure out how to add a format check to our CI...

Reply:
Yea sorry about that, I'm using the `ts-lsp` in emacs. The clang-lsp has always lined up with your formatting setup without any tweaking for me, but the js formatting has other ideas on my end.